src/meta_schedule/space_generator/space_generator.cc

```diff
@@ -21,6 +21,28 @@
 namespace tvm {
 namespace meta_schedule {
 
+static inline bool HasFlag_(String attr, std::string flag) {
+  std::string attr_str = attr;
+  return attr_str.find(flag) != std::string::npos;
+}
+
+// static inline bool HasFlag_(Optional<String> attr, std::string flag) {
+//   if (!attr) {
+//     return false;
+//   }
+//   return HasFlag_(attr.value(), flag);
+// }
+
+static inline bool HasFlag_(Optional<Array<String>> attr, std::string flag) {
```

Comment:
Looks like we have the same code in `src/target/parsers/aprofile.cc`.
Instead of code duplication, can we move it into common place?

Reply:
Code duplication fixed, we are now using different method to pull specific keys from the target.